tests/conftest.py

```diff
@@ -357,13 +358,11 @@ def factory(
                 interface_type=interface,
             )
             assert pod_obj, "Failed to create PVC"
-            helpers.wait_for_resource_state(pod_obj, constants.STATUS_RUNNING)
-            pod_obj.reload()
+            instances.append(pod_obj)
```

Comment:
If you will move indent 1 tab to the left here you can remove one from line 351 an will have the same behavior and 1 line less ;)

Reply:
done